src/worker.rs

```diff
@@ -45,6 +49,46 @@ pub struct Worker {
     logger: Logger,
 }
 
+fn get_git_sha(
+    revision: i32,
```

Comment:
Just for consistency, let's make the order of arguments here, `logger`, `revision_mapping`, `revision`, `svn_path`.